istioctl/cmd/describe_test.go

```diff
@@ -260,6 +369,36 @@ DestinationRule: ratings for "ratings"
    Traffic Policy TLS Mode: ISTIO_MUTUAL
 Pilot reports that pod is PERMISSIVE (enforces HTTP/mTLS) and clients speak mTLS
 RBAC policies: ratings-reader
+`,
+		},
+		{ // case 6 has 1.3 data, and a service with unnamed port
+			execClientConfig: cannedConfig,
+			configs:          cannedIstioConfig,
+			k8sConfigs:       cannedK8sEnv,
+			args:             strings.Split("-n default experimental describe pod productpage-v1-7bbd79f8fd-k6j79", " "),
+			expectedOutput: `Pod: productpage-v1-7bbd79f8fd-k6j79
+   Pod Ports: 15090 (istio-proxy)
+--------------------
+Service: productpage
+   Port:  9080/UnsupportedProtocol
+   9080 is named "" which does not follow Istio conventions
+Authn: None
```

Comment:
> `9080 is named ""`

Port 9080 is unnamed? 

Also, starting in 1.3, is it still convention to name ports if automatic port detection is available?

Reply:
Do you think it is worthwhile to special-case ` named ""` as `unnamed`?  It isn't hard....

Lin told me that protocol detection was not enabled for inbound in 1.3.0.  If that is the case, the 1.3.0 client should not assume port detection is available.  When there is logic for it, I will replace the current `return false` with code specific to a version or feature flag.

Reply:
Yes, probably worth it, particularly if its easy to add.